target/linux/ath79/image/generic.mk

```diff
@@ -694,6 +694,19 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/mikrotik_routerboard-wap-g-5hact2hnd
+  $(Device/mikrotik)
+  ATH := qca9556
+  DEVICE_MODEL := RouterBOARD wAP G-5HacT2HnD (wAP AC)
+  IMAGE_SIZE := 16000k
+  IMAGE/sysupgrade.bin := append-kernel | kernel2minor -s 1024 -e | \
+    pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | \
```

Comment:
Nitpick: It is common to use a single tab for hanging indent here.

Reply:
True.